brands/office/insurance.json

```
@@ -725,5 +725,15 @@
       "name:fa": "بیمه ایران",
       "office": "insurance"
     }
+  },
+   "office/insurance|ENSA": {
+    "countryCodes": ["ao"],
+    "tags": {
+      "brand": "ENSA – Seguros de Angola S.A.",
```

Just "ENSA" is probably fine here.
```suggestion
      "brand": "ENSA",
```